content/llm/api-v2.md

```
@@ -6251,6 +6337,78 @@ Reference documentation for all webhook and callback payload structures is avail
+### Source: ./content/docs/api-v2/reference/webhooks/botwebhookchatmessage.mdx
+
+
+
+
+## Payload Structure
+
+| Field | Type | Required | Description |
+|-------|------|----------|-------------|
+| `event` | string | Yes | The webhook event type |
+| `data` | object | Yes |  |
+| `extra` | object | null | Yes | Additional metadata provided when creating the bot. This is user-defined data that can be used for correlation or tracking |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)